### DISCUSSION TITLE
Any plans for "live" data

### DISCUSSION THREAD

**Discussion by brettfk** [amberelectric/public-api#84] [Q&A]
Greetings!

Are there any plans to update the API to allow for "live" data? In my case I would like to get updates every 30 mins as to what I am putting back into the grid from my solar, as opposed to waiting for the next day, if that's possible.

I'd be curious to know if this is more because the data doesn't reach the retailer for a period due to the wholesaler or if it's just something not yet supported?

Thanks!

**Comment by madpilot** [comment]
Unfortunately, we as the retailer can't get data from the meter providers in real-time, without specific agreements with each meter provider. We are trialling real-time data with one of our meter providers, but that isn't being pushed through to the API _yet_

Email info@amber.com.au with your details, and I'll see which meter provider you are on.

Outside of that - what sort of inverter do you have? It's generally easier to pull the data from the inverter if possible. The other option is a third-party device like the Rainforest Eagle 200, which connects to your smart meter using ZigBee and provides the info you want in real-time (I have one of these, and feed it into Home Assistant)

**Reply by brettfk** [reply]
Thanks madpilot 😄 
I had a suspicion that might be the case, thanks for letting me know.

Our inverter is a Solis 4th Gen, I'd have to get the exact model when I'm back at home, but after checking it seems it does not give us feed-in information (hind sight is a wonderful thing!). We already have the inverter itself in Home Assistant and I am about to install a Home Assistant Glow meter reader to give us our grid usage, but was hoping to plug the gap with what we're putting back into the grid.

What details did you need from me regarding the meter provider?

Brett

**Comment by madpilot** [comment]
Either the brand of Smart Meter, NMI or address. But email it to the above email, and I can look at it.
